packages/components/src/components/hds/app-footer/link.ts

```diff
@@ -8,11 +8,12 @@ import Component from '@glimmer/component';
 import type { HdsInteractiveSignature } from '../interactive/';
 import type { HdsLinkColors, HdsLinkIconPositions } from '../link/types.ts';
 import type { HdsLinkInlineSignature } from '../link/inline.ts';
+import type { IconName } from '@hashicorp/flight-icons/svg/index';
```

Comment:
Question: should we instead import the signature from the `EmberFlightIcon` component, and then use its `FlightIconSignature['Args']['name']?